nixpkgs_review/cli/__init__.py

```diff
@@ -223,6 +223,13 @@ def common_flags() -> List[CommonFlag]:
             default=read_github_token(),
             help="Github access token (optional if request limit exceeds)",
         ),
+        CommonFlag(
+            "--build-graph",
+            type=str,
+            default=nix_nom_tool(),
+            choices=["nix", "nom"],
+            help='Build graph to use. Use either "nom" or "nix". Will default to "nom" if available',
```

Comment:
For me --build-graph sounds more like a boolean option which when passed to nixpkgs-review would trigger to use nom. The help is not that fitting for me.

Reply:
If this isn't helpful, I'd appreciate a suggestion. 
For me, the initial implementation using `--nom` was a boolean. `--build-graph` (to me) doesn't sound like a boolean. I'd rather expect `--use-build-graph` or similar to be a boolean. But I agree, it isn't optimal, yet.
If you have a suggestion how to phrase it more distinctly, please let me know.

Reply:
I think it's fine, if you use `--help` you get this:

```
  --build-graph {nix,nom}
                        Build graph to use. Use either "nom" or "nix". Will default to "nom" if available
```